subhub/tests/unit/stripe/charge/test_stripe_charge.py

```diff
@@ -27,9 +27,8 @@ def test_stripe_webhook_succeeded(mocker):
     }
 
     # using mockito
-    mockito.when(requests).post(CFG.SALESFORCE_BASKET_URI, data=data).thenReturn(
-        response
-    )
+    basket_url = basket_url = CFG.SALESFORCE_BASKET_URI + CFG.BASKET_API_KEY
```

Comment:
double assignment? looks to be copy pasted below

Reply:
addressed in update

Reply:
again doesn't seem resolved